run_tests.py

```diff
@@ -1,14 +1,21 @@
-import sys
 import os
-import pep8ext_naming
 import re
+import sys
+
+import flake8
+from flake8.options.manager import OptionManager
```

Comment:
You should not be importing this. It's a private implementation detail of Flake8 that may change location.